src/lib/utils/classNames.js

```diff
@@ -0,0 +1,8 @@
+export const classNames = (...classes) => {
```

Comment:
8 lines of code instead of [one dependency](https://github.com/JedWatson/classnames)! 🎉 I'm curious how far we get with it.